INSTALL_ASU.md

```diff
@@ -0,0 +1,56 @@
+# General
+
+Start with the general [Enterprise new user setup](https://docs.google.com/document/d/1DOoiYEZd15KMw4KNSIkaGnph2aARI4n77cEuQHiOJqs/edit), and do the steps up to and including anaconda installation. When you create your environment, **be sure to use python 2**, not 3 as is the default. Give an appropriate name to the conda environment. Below we use `epic` as the example.
+
+After the installation of anaconda do the following
+```
+$ user=`whoami`
+$ env=epic
+ $ conda create -n ${env} python=2.7
+$ conda activate ${env}
+$ conda install ipython
+$ python -m ipykernel install --user --name ${env} --display-name ${env}
+$ pip install lsl
+```
+
+# Bifrost
+Until some pull requests are merged, we need to use a hybrid of two bifrost forks.
```

Comment:
Note James' romein branch has been merged in! We can update this to just use LEDA bifrost, but as of writing, we still need to use master + plugin-wrapper because the latter hasn't been updated.